src/server/pfs/server/driver_file.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/gogo/protobuf/proto"
```

Comment:
Was this intentional.  Weren't we trying to move away from gogo because it's unmaintained?

Reply:
this was intentional. Right now, we are 99% using gogoproto. When we migrate, we'll want to do so to the updated google backed proto lib which is incompatible with the previous. I removed this because that module ("github.com/golang/protobuf/proto") is deprecated